Editor/Resources/S3/BucketAccessPolicy.json

```diff
@@ -0,0 +1,24 @@
+﻿{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "S3ServerAccessLogsPolicy",
+      "Effect": "Allow",
+      "Principal": {
+        "Service": "logging.s3.amazonaws.com"
+      },
+      "Action": [
+        "s3:PutObject"
+      ],
+      "Resource": "arn:aws:s3:::{0}/{1}*",
```

Comment:
Are china regions supported by the plugin? I do see them in `AwsRegionMapper`so I'm guessing they are? If that's the case we should parameterize the partition based on region too.

For example in china this will be: 

```
arn:aws-cn:s3:::{0}/{1}*",
```

Reply:
So maybe use something like

```
aws:{0}:s3:::{1}/{2}*
```

Reply:
Ignore this, looks like the existing templates haven't been built for China so going to assume this isn't required. https://github.com/aws/amazon-gamelift-plugin-unity/blob/main/Editor/Resources/CloudFormation/scenario2_single_fleet/cloudformation.yml#L115C11-L115C86